src/pybind/mgr/status/module.py

```diff
@@ -137,12 +161,26 @@ def handle_fs_status(self, cmd):
                 metadata = self.get_metadata('mds', daemon_info['name'])
                 mds_versions[metadata.get('ceph_version', "unknown")].append(daemon_info['name'])
 
-                rank_table.add_row([
-                    "{0}-s".format(daemon_info['rank']), "standby-replay",
-                    daemon_info['name'], activity,
-                    mgr_util.format_dimless(dns, 5),
-                    mgr_util.format_dimless(inos, 5)
-                ])
+                if output_format in ('json', 'json-pretty'):
+                    json_output['mdsmap'].append(
+                        {
+                            'rank': rank,
+                            'name': daemon_info['name'],
+                            'state': 'standby-replay',
+                            'Evts': mgr_util.format_dimless(
+                                self.get_rate("mds", daemon_info['name'], "mds_log.replayed"),
+                                5, False),
+                            'dns': self.format_dimless(dns, 5, False),
+                            'inos': self.format_dimless(inos, 5, False)
+                        }
+                    )
```

Comment:
and move `)` to the end of previous line.  the same applies to other places in your change.